src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -94,46 +94,41 @@ export interface ShippingProps extends SystemContextProps {
   isCommittingMutation: boolean
 }
 
-export interface ShippingState {
-  shippingOption: CommerceOrderFulfillmentTypeEnum
-  address: Address
-  phoneNumber: PhoneNumber
-  phoneNumberError: PhoneNumberError
-  phoneNumberTouched: PhoneNumberTouched
-  addressErrors: AddressErrors | {}
-  addressTouched: AddressTouched
-  selectedAddressID: string
-  saveAddress: boolean
-  shippingQuotes: ShippingQuotesType
-  shippingQuoteId?: string
-  savedAddressID?: string
-}
-
 const logger = createLogger("Order/Routes/Shipping/index.tsx")
-@track()
-export class ShippingRoute extends Component<ShippingProps, ShippingState> {
-  state: ShippingState = {
-    shippingOption: getShippingOption(
-      this.props.order.requestedFulfillment?.__typename
-    ),
-    // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-    address: startingAddress(this.props.me, this.props.order),
-    addressErrors: {},
-    addressTouched: {},
-    savedAddressID: undefined,
-    phoneNumber: startingPhoneNumber(this.props.me, this.props.order),
-    phoneNumberError: "",
-    phoneNumberTouched: false,
-    selectedAddressID: defaultShippingAddressIndex(
-      this.props.me,
-      this.props.order
-    ),
-    shippingQuotes: getShippingQuotes(this.props.order),
-    shippingQuoteId: getSelectedShippingQuoteId(this.props.order),
-    saveAddress: true,
-  }
 
-  get touchedAddress() {
+export const ShippingRoute: React.FC<ShippingProps> = props => {
```

Comment:
nit: you can destructure the props here and this would save some verbosity along the component of calling `props.something`.